public/docs/swagger.yaml

```diff
@@ -516,6 +516,26 @@ paths:
         - "Routes"
       operationId: createRoute
       summary: Create a route.
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/Create_route'
+        401:
+          description: Unauthorized request.
+          schema:
+            $ref: '#/definitions/Error'
+        500:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/Error'
+      security:
+        - ApiKey: []
+    get:
+      tags:
+        - "Routes"
+      operationId: listingRoutes
+      summary: Listing all the routes from the organization.
       responses:
         200:
```

Comment:
Here you are not listing the routes in an array, as in  200 OK responses in other GET services